<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dell-i37-9Z1F2P2-w11x64" failures="2" tests="7" name="Test" time="0.156" errors="0" timestamp="2024-09-17T14:14:06 IST">
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test1" time="0.026">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test1" time="0.003">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test1" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests.test1(MyTests.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test2" time="0.004">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test2" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests.test2(MyTests.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="ReRun_Failed_TCs_usingIAnnotationTransformer.MyTests" name="test3" time="0.002"/>
</testsuite> <!-- Test -->
